wagtail/compat.py

```diff
@@ -11,3 +12,5 @@
     raise ImproperlyConfigured(
         "AUTH_USER_MODEL must be of the form" " 'app_label.model_name'"
     )
+
+BEAUTIFUL_SOUP_FEATURES = [FAST, HTML]
```

Comment:
Thought: Instead of having BS4 work it out, we could detect `lxml` and modify the feature list here instead? This would avoid the need to silence a warning.